client/src/main/java/io/confluent/kafka/schemaregistry/client/LocalSchemaRegistryClient.java

```diff
@@ -156,4 +163,50 @@ public synchronized int getVersion(String subject, Schema schema)
       throw new IOException("Cannot get version from schema registry!");
     }
   }
+  
+  @Override
+  public boolean testCompatibility(String subject, Schema newSchema) throws IOException,
+      RestClientException {
+    SchemaMetadata latestSchemaMetadata = getLatestSchemaMetadata(subject);
+    Schema latestSchema = getSchemaByIdFromRegistry(latestSchemaMetadata.getId());
+    String config = configCache.getOrDefault(subject, DEFAULT_COMPATIBILITY);
+    SchemaValidator validator = null;
```

Comment:
map.getOrDefault() is a java 8 feature. Currently, schema registry needs to support java 6.